aspnetcore/release-notes/aspnetcore-5.0.md

```diff
@@ -0,0 +1,480 @@
+
+<!-- WAIT ON THIS ENTRY UNTIL https://github.com/dotnet/AspNetCore.Docs/pull/19887 MERGES.
+
+### Component integration across hosting models
+
+Component integration is improved across hosting models.
+
+For more information, see <xref:blazor/components/integrate-components-into-razor-pages-and-mvc-apps>.
+
+-->
+
+### Linker is now a Trimmer
+
+Prior to the release of ASP.NET Core 5.0, Blazor WebAssembly performs Intermediate Language (IL) linking during a build to trim unnecessary IL from the app's output assemblies. After the release of 5.0, Blazor WebAssembly performs *trimming* to reduce the size of the published output. For more information, see <xref:blazor/host-and-deploy/configure-trimmer>.
+
+### Browser compatibility analyzer
+
+Blazor WebAssembly apps target the full .NET API surface area, but not all .NET APIs are supported on WebAssembly due to browser sandbox constraints. Unsupported APIs throw <xref:System.PlatformNotSupportedException> when running on WebAssembly. A platform compatibility analyzer warns the developer when the app uses APIs that aren't supported by the app's target platforms. For more information, see <xref:blazor/components/class-libraries#browser-compatibility-analyzer-for-blazor-webassembly>.
+
+### Lazy load assemblies
+
+Blazor WebAssembly app startup performance can be improved by deferring the loading of some application assemblies until they are required. For more information, see <xref:blazor/webassembly-lazy-load-assemblies>.
+
+### Updated globalization support
+
+Globalization support is available for Blazor WebAssembly based on International Components for Unicode (ICU). For more information, see <xref:blazor/globalization-localization>.
+
+## gRPC
+
+Many preformance improvements have been made in [gRPC](https://grpc.io/). <!-- For more information, see [this ]
+Waiting for https://github.com/aspnet/specs/pull/523
+--> 
+For more information, see <xref:grpc/index>.
+
+## SignalR
+
+SignalR Hub filters, called Hub pipelines in ASP.NET SignalR, is a feature that allows code to run before and after Hub methods are called. Running code before and after Hub methods are called is similar to how middleware has the ability to run code before and after an HTTP request. Common uses include logging, error handling, and argument validation.
+
+For more information, see [Use hub filters in ASP.NET Core SignalR](xref:signalr/hub-filters).
+
+### SignalR parallel hub invocations
+
+ASP.NET Core SignalR is now capable of handling parallel hub invocations. The default behavior can be changed to allow clients to invoke more than one hub method at a time:
+
+[!code-csharp[](~/release-notes/sample/StartupSignalRhubs.cs?name=snippet)]
+
+### Added Messagepack support in SignalR Java client
+
+<!-- Review: I can't find NuGet com.microsoft.signalr.messagepack  -->
```

Comment:
This is for Java, so it's not a NuGet package. https://mvnrepository.com/artifact/com.microsoft.signalr.messagepack